web3.js/test/connection.test.ts

```diff
@@ -77,6 +82,23 @@ const verifySignatureStatus = (
   return status;
 };
 
+class TransactionExpiredBlockheightExceededError extends Error {
+  signature: string;
+
+  constructor(signature: string) {
+    super(`Signature ${signature} has expired.`);
+    this.signature = signature;
+  }
+}
+
+Object.defineProperty(
+  TransactionExpiredBlockheightExceededError.prototype,
+  'name',
+  {
+    value: 'TransactionExpiredBlockheightExceededError',
+  },
+);
+
```

Comment:
Did it not work to just `import` this error type?

Reply:
Whoops, I forgot to update this.